docker-compose.override.yaml

```diff
@@ -2,14 +2,26 @@
 # to provide dependencies and expose ports for local testing
 
 services:
-  ffc-template-node:
+  ffc-pay-batch-verifier:
     build:
       target: development
-    image: ffc-template-node-development
-    container_name: ffc-template-node-development
+    depends_on:
+      - ffc-pay-batch-azurite
+    image: ffc-pay-batch-verifier-development
+    container_name: ffc-pay-batch-verifier-development
     ports:
-      - "3000:3000"
-      - "9229:9229"
+      - "9260:9229"
     volumes:
       - ./app:/home/node/app
       - ./package.json:/home/node/package.json
+    environment:
+      AZURE_STORAGE_CONNECTION_STRING: DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://ffc-pay-batch-azurite:10000/devstoreaccount1;
```

Comment:
Just a question: why do we use DEV storage account rather than SND?

Reply:
`devstoreaccount` and that account key are the defaults setup in the Azurite storage emulator.  So anytime we want to connect to Azurite you'll see the same connection string.  Only with a different hostname at the end.

![image](https://user-images.githubusercontent.com/42574977/198238419-90025b53-eb84-4d38-8c69-3db772c7fdc7.png)

https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub